pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   path_provider: ^1.6.21
   args: ^1.6.0
   translator: ^0.1.5
```

Comment:
TODO: If we will not use this library "translator: ^0.1.5" with unofficial google API, then it is better to delete it before publishing

Reply:
We still use it as an option. See the first option in the map below:

Map<String, String> _providers = {
  "GoogleTest": "Google API through a back door",
  "Google": "Google using API key",
};